src/org/labkey/test/components/ui/files/ImageFileViewDialog.java

```diff
@@ -0,0 +1,37 @@
+package org.labkey.test.components.ui.files;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ImageFileViewDialog extends ModalDialog
+{
+    public ImageFileViewDialog(WebDriver driver, String filename)
+    {
+        super(new ModalDialogFinder(driver).withTitle(filename).waitFor().getComponentElement(), driver);
+    }
+
+    public boolean isImageRendered()
+    {
+        return elementCache().img.isDisplayed();
```

Comment:
Would there be value in passing in a string value that is the image file name, or some part of it, and then validating that not only is the element displayed but the correct file was used?